.github/workflows/new-cfe-notifications.yml

```diff
@@ -26,49 +55,65 @@ jobs:
           slack-optional-unfurl_media: false
         continue-on-error: true
 
-
   cfe-approved:
     if: github.event.label.name == 'CFE Approved'
     runs-on: ubuntu-20.04
+    needs:
+      - prep
+    permissions:
+      pull-requests: write
+      issues: write
     steps:
-      - name: Extract PR URL from issue body
+      - name: Extract PR number from issue body
         id: extract-pr
-        run: |
-          PR_URL=$(echo "${{ github.event.issue.body }}" | grep -oP 'https://github.com/[^/]+/[^/]+/pull/\d+')
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const event = ${{ toJSON( github.event ) }}
```

Comment:
Similar to what was mentioned before, maybe we can use directly `github.event.issue.body` if possible.